src/profile/external-identities/FenceAccount.ts

```diff
@@ -24,7 +25,9 @@ export const FenceAccount = ({ provider: { key, name, expiresAfter, short } }) =
   const [isLinking, setIsLinking] = useState(Nav.useRoute().name === 'fence-callback' && key === provider);
 
   // Helpers
-  const redirectUrl = `${window.location.origin}/${Nav.getLink('fence-callback')}`;
+  const redirectUrl = `${
+    window.location.hostname === 'localhost' ? getConfig().devUrlRoot : window.location.origin
```

Comment:
I need to make a stink about "do x on my machine" patterns like this... this feels hacky, and assumes pointing to dev is the one best option for local setup.

Reply:
This copies whats done in `PassportLinker`, and is really the only way we can test this flow without deploying to Dev. I'll remove it for now, but we should solve this.

Reply:
I'd suggest adding the redirect URL to the config.json files, where it can be overridden with `window.configOverridesStore`.

Reply:
Never mind, that won't work because of the different branded sites. But still, could use `configOverridesStore` to override the default behavior.

Reply:
Removed. We can tackle this when we combine all OAuth links into a single component.